src/world.rs

```diff
@@ -0,0 +1,190 @@
+use std::{
+    io::Read,
+    path::{Path, PathBuf},
+};
+
+use regex::Regex;
+use serde::Deserialize;
+
+use crate::{Error, ResourceReader};
+
+/// A World is a list of maps files or regex patterns that define a layout of TMX maps.
+/// You can use the loader to further load the maps defined by the world.
+#[derive(Deserialize, PartialEq, Clone, Debug)]
+pub struct World {
+    /// The path first used in a [`ResourceReader`] to load this world.
+    #[serde(skip_deserializing)]
+    pub source: PathBuf,
```

Comment:
Hi, sorry to interfere but it seems like this field is never set. At least in my testings it's always empty.
Would be nice to have the asset source as it is done for maps.

Reply:
Good catch. I've fixed that in the latest commit and made sure it's checked in the test.

Thanks!